<?xml version="1.0"?>
<launch>

  <arg name="map_width" default="23" />
  <arg name="map_height" default="23" />

	<!-- TODO add any calls to your code -->

  <node pkg="simple_control" type="global_planner.py" name="global_planner" output="screen">
    <param name="map_width" type="int" value="$(arg map_width)" />
    <param name="map_height" type="int" value="$(arg map_height)" />
  </node>

</launch>